textproc/libiconv/Portfile

```diff
@@ -39,18 +39,11 @@ set myprefix ${prefix}
 
 # This port is used by clang-3.4 to bootstrap libcxx
 subport ${name}-bootstrap {
-    set myprefix ${prefix}/libexec/libcxx-bootstrap
```

Comment:
Use of `myprefix` needs to be cleaned up in this Portfile too.

Reply:
Done